ai_genomics/pipeline/entity_extraction/postprocessing_pipeline.py

```diff
@@ -0,0 +1,63 @@
+from ai_genomics.pipeline.entity_extraction.postprocess_entities import EntityCleaner
+from ai_genomics.getters.data_getters import save_to_s3
+
+from metaflow import FlowSpec, step, retry
+
+import ast
+import boto3
+import json
+
+s3 = boto3.resource("s3")
```

Comment:
you can also import this from ai_genomics.getters.data_getters, i.e:

`from ai_genomics.getters.data_getters import S3`